File: book-network-ui/src/app/services/models/user-response.ts
/* tslint:disable */
/* eslint-disable */
import { Book } from '../models/book';
import { Role } from '../models/role';
export interface UserResponse {
  bio?: string;
  books?: Array<Book>;
  createdAt?: string;
  email?: string;
  firstName?: string;
  id?: number;
  lastName?: string;
  profilePicture?: Array<string>;
  roles?: Array<Role>;
  updatedAt?: string;
}

File: book-network-api/src/main/java/com/dev/kei/book/network/api/user/UserResponse.java
package com.dev.kei.book.network.api.user;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Builder
public class UserResponse {
    private Long id;
    private String firstName;
    private String lastName;
    private String email;
    private String bio;
    private byte[] profilePicture;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private String updatedJWT;
}
